trivy.log

```diff
@@ -0,0 +1,12 @@
+2020-02-14T22:56:07.711+0100	[34mINFO[0m	Detecting Alpine vulnerabilities...
```

Comment:
Same question with this log file. Is it needed or can we add to .gitignore?

Reply:
Will remove